sycl/source/detail/scheduler/commands.cpp

```diff
@@ -70,6 +70,20 @@ bool CurrentCodeLocationValid() {
 struct DemangleHandle {
   char *p;
   DemangleHandle(char *ptr) : p(ptr) {}
+
+  DemangleHandle &operator=(const DemangleHandle &rhs) {
```

Comment:
Same question here, we don't have any assignment in the code, shouldn't it be deleted (= delete)?

Reply:
That doesn't answer the question why we can't use ` = delete;` instead.